frame/staking/src/pallet/impls.rs

```diff
@@ -983,6 +1008,23 @@ impl<T: Config> Pallet<T> {
 			DispatchClass::Mandatory,
 		);
 	}
+
+	/// Temporary getter for `Payees`.
+	///
+	/// Migrates `Payee` to `Payees` if it has not been migrated already.
+	pub fn get_payout_destination(
```

Comment:
does it need to be pub?

Reply:
Yes It is being tested solo, wanted to ensure that it is successfully migrating.